web/packages/teleport/src/Discover/Shared/Aws/StatusCell.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Flex } from 'design';
+import React from 'react';
+
+import { DisableableCell as Cell } from 'teleport/Discover/Shared';
+
+import { StatusLight, ItemStatus } from '../StatusLight';
+
+export const StatusCell = ({
+  status,
+  statusText,
+  disabled,
+  disabledText,
+}: {
+  status: ItemStatus;
+  statusText: string;
+  disabled: boolean;
+  disabledText: string;
+}) => {
+  return (
+    <Cell disabled={disabled} disabledText={disabledText}>
+      <Flex alignItems="center">
```

Comment:
```suggestion
      <Flex alignItems="baseline">
```

This makes the status light actually aligned with the text.